### PR TITLE
Fix the metadata

### PR DESCRIPTION
**Pull request by open-junius**
## Description
The pr fixed the metadata via write the comments in correct way. It adds an unit test in runtime to verify.

## Related Issue(s)

- Closes #[384]

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Other (please describe):

## Breaking Change

If this PR introduces a breaking change, please provide a detailed description of the impact and the migration path for existing applications.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have run `cargo fmt` and `cargo clippy` to ensure my code is formatted and linted correctly
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots (if applicable)

Please include any relevant screenshots or GIFs that demonstrate the changes made.

## Additional Notes

Please provide any additional information or context that may be helpful for reviewers.

part of #384

### CODE REVIEW

File: runtime/tests/pallet_proxy.rs [hunk 1]
```diff
@@ -184,18 +184,18 @@ fn test_proxy_pallet() {
...
-                    AccountId::from(DELEGATE).into(),
-                    *proxy_type,
-                    0
-                ));
-
-                verify_call_with_proxy_type(proxy_type, &call());
-            });
-        }
-    }
+    // for call in calls.iter() {
```
Comment (distributedstatemachine) [inline runtime/tests/pallet_proxy.rs]: Why is this commented out ? If is not needed anymore would it make sesne to delete it ?

File: runtime/tests/pallet_proxy.rs [hunk 2]
```diff
@@ -172,7 +172,7 @@ fn test_proxy_pallet() {
         ProxyType::Registration,
     ];
 
-    let calls = [
+    let _calls = [
```
Comment (distributedstatemachine) [inline runtime/tests/pallet_proxy.rs]: Also this , its not being used anymore ?

File: runtime/tests/pallet_proxy.rs (hunk 1 above)
Comment (open-junius) [reply runtime/tests/pallet_proxy.rs]: thanks for review. the PR is in progress, not ready for review yet. I set it as in progress, maybe I should set it as draft. but I am not sure if CI works for draft PR.

**Review by orriin — APPROVED** [review]
Awesome!

**Pull request merged**